### PR TITLE
Testing ci

### PR DESCRIPTION
**Pull request by Skullbock**

### CODE REVIEW

File: libraries/zoolanders/Framework/Request/MockRequest.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    /**
+     * Sets a value
+     *
+     * @param   string  $name   Name of the value to set.
+     * @param   mixed   $value  Value to assign to the input.
+     *
+     * @return  void
+     *
+     * @since   11.1
```
Comment (miljan-aleksic) [inline libraries/zoolanders/Framework/Request/MockRequest.php]: to be removed, I assume

File: libraries/zoolanders/Framework/Request/RequestInterface.php [hunk 2]
```diff
@@ -64,4 +64,16 @@ public function getHttpVerb ();
...
+
+    /**
+     * Sets a value
+     *
+     * @param   string  $name   Name of the value to set.
+     * @param   mixed   $value  Value to assign to the input.
+     *
+     * @return  void
+     *
+     * @since   11.1
```
Comment (miljan-aleksic) [inline libraries/zoolanders/Framework/Request/RequestInterface.php]: as well

File: package.json [hunk 3]
```diff
@@ -11,6 +11,8 @@
   },
   "scripts": {
     "build": "node build/build.js",
+    "build-1": "node build/build-1.js",
```
Comment (miljan-aleksic) [inline package.json]: to be removed

File: libraries/zoolanders/Framework/Request/MockRequest.php (hunk 1 above)
Comment (Skullbock) [inline libraries/zoolanders/Framework/Request/MockRequest.php]: done

File: libraries/zoolanders/Framework/Request/RequestInterface.php (hunk 2 above)
Comment (Skullbock) [inline libraries/zoolanders/Framework/Request/RequestInterface.php]: done

File: package.json (hunk 3 above)
Comment (Skullbock) [inline package.json]: done

**Pull request merged**